FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -4,9 +4,20 @@
 
 import Foundation
 
+public struct FeedImageListRemote: Decodable {
```

Comment:
`FeedImageListRemote` is a private implementation detail and it should not be visible from outside of the module. Moreover there's no need to make it internal as this type is only accessed inside `RemoteFeedLoader.swift`. Thus it can be private
Please make `FeedImageListRemote` private

```suggestion
private struct FeedImageListRemote: Decodable {
```